src/js/controllers/navbar.controller.js

```diff
@@ -54,9 +50,15 @@ module.controller('NavbarCtrl', ['$scope', '$element', '$compile', 'rpId',
       $scope.message = 'Connected to the Ripple network. Fee: ' + $scope.currentFee + ' XRP';
     }
     else {
-      $scope.message = 'Connected to the Ripple network.';
+      $scope.message = 'Connected to the Ripple network.'; 
```

Comment:
@vhpoet Because the messages have to do with logic in the controller, I thought I should keep everything in the same place. I think we had talked about this before too. But what do you think is a better way of doing it?